### PR TITLE
Updated to use the new base_path

### PR DESCRIPTION
**Pull request by mkanoor**
New gem based on 
https://github.com/ManageIQ/topological_inventory-api/pull/158

### CODE REVIEW

**Comment by mkanoor** [comment]
@agrare @Fryguy 
Please review
Generated using openapi-generator

File: .rubocop.yml [hunk 1]
```diff
@@ -1,3 +1,154 @@
-inherit_from:
-- https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop_base.yml
-- .rubocop_local.yml
+# This file is based on https://github.com/rails/rails/blob/master/.rubocop.yml (MIT license)
```
Comment (Fryguy) [inline .rubocop.yml]: This file is not going to play nice with the bot.  Can you add the exclusions to rubocop.local.yml, and then mark this file as not to be touched in the autogenerator?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,5 @@
 # topological_inventory-api-client
 
-[![Build Status](https://travis-ci.org/ManageIQ/topological_inventory-api-client-ruby.svg?branch=master)](https://travis-ci.org/ManageIQ/topological_inventory-api-client-ruby)
-[![Maintainability](https://api.codeclimate.com/v1/badges/ae0a09f704877cb2ff80/maintainability)](https://codeclimate.com/github/ManageIQ/topological_inventory-api-client-ruby/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/ae0a09f704877cb2ff80/test_coverage)](https://codeclimate.com/github/ManageIQ/topological_inventory-api-client-ruby/test_coverage)
-[![Security](https://hakiri.io/github/ManageIQ/topological_inventory-api-client-ruby/master.svg)](https://hakiri.io/github/ManageIQ/topological_inventory-api-client-ruby/master)
-
```
Comment (Fryguy) [inline README.md]: Similar.  Please don't undo some of these changes.

File: README.md [hunk 3]
```diff
@@ -233,7 +247,3 @@ Class | Method | HTTP request | Description
 
 - **Type**: HTTP basic authentication
 
-## License
-
-This project is available as open source under the terms of the [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0).
-
```
Comment (Fryguy) [inline README.md]: Same.

File: topological_inventory-api-client.gemspec [hunk 4]
```diff
@@ -37,7 +37,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'autotest-rails-pure', '~> 4.1', '>= 4.1.2'
   s.add_development_dependency 'autotest-growl', '~> 0.2', '>= 0.2.16'
   s.add_development_dependency 'autotest-fsevent', '~> 0.2', '>= 0.2.12'
-  s.add_development_dependency "simplecov"
```
Comment (Fryguy) [inline topological_inventory-api-client.gemspec]: Same.

File: .gitignore [hunk 5]
```diff
@@ -5,7 +5,6 @@
 *.rbc
 /.config
 /coverage/
-/Gemfile.lock
```
Comment (Fryguy) [inline .gitignore]: We don't want to commit the Gemfile.lock.

**Pull request merged**